docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-db-query-logging.md

```diff
@@ -0,0 +1,124 @@
+---
+title: ybm CLI cluster db-query-logging
+headerTitle: ybm cluster db-query-logging
+linkTitle: cluster db-query-logging
+description: YugabyteDB Aeon CLI reference Cluster Database Query Logging Resource.
+headcontent: Manage cluster database query logging
+menu:
+  preview_yugabyte-cloud:
+    identifier: managed-cli-db-query-logging
+    parent: managed-cli-reference
+    weight: 20
+type: docs
+---
+
+Use the `cluster db-query-logging` resource to perform operations on a YugabyteDB Aeon cluster, including the following:
+
+- enable, disable, and update db query logging
+- get information about db query logging
+
+## Prerequisite
+
+Before using these commands, you must have an [integration configuration](../../../cloud-monitor/logging-export.md/#prerequisites) already set up. This configuration defines the authentication and connection details for the third-party tool where logs will be exported.
+
+## Syntax
+
+```text
+Usage: ybm cluster db-query-logging [command] [flags]
+```
+
+## Examples
+
+Enable a database query-logging for a cluster:
+
+```sh
+ybm cluster db-query-logging enable \
+--cluster-name your-cluster \
+--integration-name your-integration \
+--log-line-prefix "%m :%r :%u @ %d :[%p] : %a :" \
+--log-min-duration-statement 30 \
+--log-connections true \
+--log-duration false \
+--log-error-verbosity DEFAULT \
+--log-statement MOD
```

Comment:
It is very likely that users will simply copy paste this command for trying out enabling logs of a cluster.
So let's keep all `--log-*` flags value set to default as mentioned [here](https://api-docs.yugabyte.com/docs/managed-apis/badf5234abfd9-create-a-database-query-logging-configuration-and-associate-with-cluster). The above command rn have some stuff enabled like log-connections, log-statement etc.

Reply:
Done